maps-app/src/main/res/values/strings.xml

```diff
@@ -26,7 +26,7 @@
     <string name="no_maps">No maps</string>
 
     <!-- webmap urls -->
-    <string name="default_basemap_url">http://www.arcgis.com/home/item.html?id=d5e02a0c1f2b4ec399823fdd3c2fdebd</string>
+    <string name="default_basemap_url">http://www.arcgis.com/home/item.html?id=26888b0c21a44eb1ba2f26d1eb7981fe</string>
 
```

Comment:
@zinfin Is this the intended web map you want to start with? Or just a test until we have created one?